pkg/antctl/raw/proxy/command_test.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2023 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package proxy
+
+import (
+	"bytes"
+	"fmt"
+	"io/fs"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"k8s.io/klog/v2"
+)
+
+func TestValidateAndComplete(t *testing.T) {
+	tempFile, err := os.CreateTemp("", "tmpfile")
```

Comment:
let's use afero (already an antrea dependency) for some cleaner code

in command.go:
```go
var fs = afero.NewOsFs()
// then use fs.Stat in validateAndComplete
```

in command_test.go:
```go
func TestValidateAndComplete(t *testing.T) {
    // ...
    fs = afero.NewMemMapFs()
    // create files / directories as needed in fs
```

there are multiple examples of this in the Antrea repo, e.g. in `pkg/apiserver/registry/system/supportbundle`.

Reply:
done.